packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -110,7 +110,7 @@ export const ArticlePostCard = forwardRef(function PostCard(
           </CardTitle>
         </CardTextContainer>
         {!showFeedback && (
-          <Container className="mb-8 tablet:mb-0">
```

Comment:
Since we have now introduced the `max-h-card` on all cards to be strict with consistency, this margin causes the unexpected expansion of the content which is not needed due to the element exactly below this (`CardSpace`).